README.md

```diff
@@ -1,82 +1,64 @@
+yarn create next-app --example https://github.com/ginetta/swiss-legal-conversational-ui
 ```
 
-```bash
-yarn create next-app --example https://github.com/brunnolou/next-langchain-rag-ollama
-```
-
-```bash
-pnpm create next-app --example https://github.com/brunnolou/next-langchain-rag-ollama
-```
 
 To run the example locally you need to:
 
-## Run Ollama
-
-1. [Download Ollama](https://ollama.ai/download) and install it locally.
-2. run `ollama run mistral` to download and install the model locally (Requires 4.1GB and 8GB of RAM)
-3. Open http://localhost:11434 to check if _Ollama is running_.
-
-## Setup Qdrant vector database
-
-1. Run `docker pull qdrant/qdrant` and `docker run -p 6333:6333 qdrant/qdrant`
-2. Open the console dashboard [`localhost:6333/dashboard#/console`](http://localhost:6333/dashboard#/console).
-3. Create a new collection running this:
-
-```
-PUT collections/swiss-or
-{
-  "vectors": {
-    "size": 384,
-    "distance": "Cosine"
-  }
-}
-```
-
-4. Download the [snapshot file](https://huggingface.co/datasets/brunnolou/swiss-code-of-obligations/resolve/main/swiss-code-of-obligations-articles-gte-small-2023-10-18-12-13-25_qdrant-v1-6-1.snapshot.zip)
-5. Unzip the file using the terminal (⚠️ **_not with Finder on Mac_** ⚠️) with `unzip ...`
-6. Upload the file using the following command. Adapt the fields accordingly and run it from the same directory, as where your snapshot lies.
-
-```
+## 🦙 Run Ollama
+
+1. [Download Ollama](https://ollama.ai/download) and install it locally
+2. `ollama run mistral` to download and install the model locally (Requires 4.1GB and 8GB of RAM)
+3. Open <http://localhost:11434> to check if _Ollama is running_
+
+## 💾 Setup Qdrant Vector Database
+
+1. `docker pull qdrant/qdrant`
+1. `docker run -p 6333:6333 qdrant/qdrant`
+1. Open the Qdrant dashboard console  <http://localhost:6333/dashboard#/console>
+1. Create a new collection running this:
+    ```curl
+    PUT collections/swiss-or
+    {
+      "vectors": {
+        "size": 384,
+        "distance": "Cosine"
+      }
+    }
+    ```
+1. Download the [snapshot file](https://huggingface.co/datasets/brunnolou/swiss-code-of-obligations/resolve/main/swiss-code-of-obligations-articles-gte-small-2023-10-18-12-13-25_qdrant-v1-6-1.snapshot.zip)
+1. Unzip the file using the terminal (⚠️ **_not with Finder on Mac_** ⚠️) with `unzip swiss-code-of-obligations-articles-gte-small-2023-10-18-12-13-25_qdrant-v1-6-1.snapshot.zip`
+1. Upload the file using the following command. Adapt the fields accordingly and run it from the same directory, as where your snapshot lies
+
+```shell
 curl -X POST 'http://localhost:6333/collections/swiss-or/snapshots/upload' \
     -H 'Content-Type:multipart/form-data' \
-    -F 'snapshot=@swiss-code-of-obligations-articles-gte-small-2023-10-18-12-13-25.snapshot'
+    -F 'snapshot=swiss-code-of-obligations-articles-gte-small-2023-10-18-12-13-25.snapshot'
```

Comment:
I think the @ is needed, isn't it? Doesn't it say that a file is being uploaded, otherwise doesn't it treat the value as a literal string?

Reply:
For me I has to remove it. @maricabertarini did you use the `@` symbol?

Reply:
I just tested. Yes, the `@` way works for me... the other doesn't; gives the following error:
`{"status":{"error":"Service internal error: File IO error: failed to iterate over archive"},"time":0.027164958}⏎    `

Reply:
I've reverted that commit.